src/Command/RunCommand.php

```diff
@@ -78,20 +78,21 @@ protected function configure(): void
                     'Name of the Test framework to use ("%s")',
                     implode('", "', TestFrameworkTypes::TYPES)
                 ),
-                ''
```

Comment:
it's now `null`, not the empty string. Is it expected change?

Reply:
Yes: in the end it does not change the result thanks to how we process the value (when we take it from the input). So although this change does not change anything downstream, I thought it had the merit of pointing "hey there's a default that depend on that" and that when refactoring this value, you would be lead to think of the default container value